### PR TITLE
Enable OV Myriad Plugin in CI

### PR DESCRIPTION
**Pull request by bani-intelaipg**
See https://buildkite.com/ngraph/models-vpu/builds/2

### CODE REVIEW

File: build_ngtf.py [hunk 1]
```diff
@@ -339,21 +339,14 @@ def main():
...
-            "-DENABLE_SAMPLES=OFF",
-            "-DENABLE_FUNCTIONAL_TESTS=OFF",
-            "-DENABLE_VPU=OFF",  # TODO: Fix OpenVINO VPU build
-            "-DNGRAPH_ONNX_IMPORT_ENABLE=OFF",
-            "-DNGRAPH_TEST_UTIL_ENABLE=OFF",
-            "-DNGRAPH_USE_CXX_ABI=" + cxx_abi,
+            "-DENABLE_V10_SERIALIZE=ON", "-DENABLE_TESTS=OFF",
+            "-DENABLE_SAMPLES=OFF", "-DENABLE_FUNCTIONAL_TESTS=OFF",
+            "-DENABLE_VPU=ON", "-DNGRAPH_ONNX_IMPORT_ENABLE=OFF",
+            "-DNGRAPH_TEST_UTIL_ENABLE=OFF", "-DNGRAPH_USE_CXX_ABI=" + cxx_abi,
```
Comment (adk9) [inline build_ngtf.py:345]: Does the code formatter do this? One per line is easier to read here.

File: test/ci/buildkite/run_pipeline.sh [hunk 2]
```diff
@@ -27,6 +27,9 @@ elif [ "${BUILDKITE_PIPELINE_NAME}" == "gpu" ]; then
 elif [ "${BUILDKITE_PIPELINE_NAME}" == "models-gpu" ]; then
    export NGRAPH_TF_BACKEND=GPU
    PIPELINE_STEPS+=" ${SCRIPT_DIR}/models-cpu.yml "
+elif [ "${BUILDKITE_PIPELINE_NAME}" == "models-vpu" ]; then
+   export NGRAPH_TF_BACKEND=MOVIDIUS
```
Comment (adk9) [inline test/ci/buildkite/run_pipeline.sh:31]: There is no plugin called `MOVIDIUS` afaik. Are you sure this is testing the right backend?

Comment (bani-intelaipg) [reply test/ci/buildkite/run_pipeline.sh:31]: I got it from here: https://docs.openvinotoolkit.org/2021.1/openvino_docs_install_guides_installing_openvino_linux.html#run-a-sample

Comment (adk9) [reply test/ci/buildkite/run_pipeline.sh:31]: Shouldn't it be `MYRIAD` according to that link?

File: build_ngtf.py (hunk 1 above)
Comment (bani-intelaipg) [reply build_ngtf.py:345]: Yes, I edited and then formatter overwrote

File: test/ci/buildkite/run_pipeline.sh (hunk 2 above)
Comment (bani-intelaipg) [reply test/ci/buildkite/run_pipeline.sh:31]: Yes, fixed

**Review by adk9 — COMMENTED** [review]
Looks good, but can you rebase against master? The OV build configuration moved to `build_utils.py` in #696.

**Review by adk9 — APPROVED** [review]

**Review by adk9 — COMMENTED** [review]
I think that we should call this pipeline `models-myriad` to be true to our naming convention.

**Pull request merged**